libs/tup-components/src/software/SoftwareTable.global.css

```diff
@@ -0,0 +1,32 @@
+@import url("@tacc/core-styles/src/lib/_imports/tools/selectors.css");
+
+/* TODO: Remove this after:
+         1. adding this to core-styles.base */
+[type="search"] {
+  outline-offset: revert; /* undo Bootstrap _reboot */
+}
+
+/* TODO: Remove this after:
+         1. adding/changing these in core-styles.base */
+.modal-content {
+  border: var(--global-border-width--normal) solid var(--global-color-primary--dark);
+}
+.modal-header {
+  align-items: center;
+}
+.modal-header .close {
+  font-size: 3.2rem;
+}
+.modal-title {
+  /* TODO: Confirm whether this is for CMS or Portal */
+  /* NOTE: Portal design looks like 1.6rem */
+  font-size: 2.1rem;
+}
```

Comment:
I have [asked designers (privately)](https://tacc-team.slack.com/archives/G01K9ET7JM7/p1679530405531209) whether this larger header text is for CMS modals or all modals.